docs/sources/configure-client/aws-lambda.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "AWS Lambda Profiling Extension"
+menuTitle: "AWS Lambda Profiling Extension"
+description: "Profiling AWS Lambda functions with Pyroscope"
+weight: 100
+---
+
+# AWS Lambda Profiling Extension
+
+## Introduction to Pyroscope Lambda Extension
+
+The Pyroscope AWS Lambda Extension is a robust tool for profiling AWS Lambda functions, ensuring minimal latency impact. This profiling is essential for optimizing your functions.
```

Comment:
We don't need the heading since this sentence acts as the introduction. It's also good practice to have a sentence between headings.

```suggestion
The Pyroscope AWS Lambda Extension is a robust tool for profiling AWS Lambda functions, ensuring minimal latency impact. This profiling is essential for optimizing your functions.
```